src/modules/app/route-configs/NewsRouteConfig.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+
+import type { AllPayloadsType } from './RouteConfig'
+import { RouteConfig } from './RouteConfig'
+import type { Route } from 'redux-first-router'
+import {
+  createCitiesEndpoint,
+  createEventsEndpoint,
+  createLocalNewsEndpoint,
+  createLanguagesEndpoint,
+  LocalNewsModel,
+  Payload
+} from '@integreat-app/integreat-api-client'
+import fetchData from '../fetchData'
+import { cmsApiBaseUrl } from '../constants/urls'
+
+type NewsRouteParamsType = {| city: string, language: string |}
+type RequiredPayloadsType = {| news: Payload<Array<LocalNewsModel>> |}
+
+export const NEWS_ROUTE = 'NEWS'
```

Comment:
I suggest to move routes constant to separate file

Reply:
We have a weak convention to put constants in files `src/modules/app/contants.js` or  `src/modules/app/contants/index.js`

Not sure whether it makes sene there though :)

Reply:
This is fine as it is, since it is consistent with existing route configs. We may have to refactor it at some point though.